### PR TITLE
MagiQuest maintenance

### PR DESCRIPTION
**Pull request by eshicks4**
Fixes to the MagiQuest encoder/decoder

### CODE REVIEW

File: src/ir_MagiQuest.cpp [hunk 1]
```diff
@@ -40,52 +41,42 @@ union magiquest_t {
...
-//    data.llword = 0;
-//    data.cmd.wand_id = wand_id;
-//    data.cmd.magnitude = magnitude;
 
     // Set IR carrier frequency
     enableIROut(38);
 
     // 2 start bits
-    sendPulseDistanceWidthData(MAGIQUEST_ONE_MARK, MAGIQUEST_ONE_SPACE, MAGIQUEST_ZERO_MARK, MAGIQUEST_ZERO_SPACE, 0, 2, PROTOCOL_IS_MSB_FIRST);
+    sendPulseDistanceWidthData(
```
Comment (ArminJo) [inline src/ir_MagiQuest.cpp:59]: I have my own formatter (K&R) so this may change.

Comment (eshicks4) [reply src/ir_MagiQuest.cpp]: that's fine.  pretty sure i left the flag on that lets the maintainer change stuff

**Comment by eshicks4** [comment]
whoops sorry - had no idea i could still modify the pull request just by pushing new changes to my fork

**Comment by ArminJo** [comment]
Thanks a lot for maintaining this protocol.
Two minor changes and it is done 😀 


> 
> 
> whoops sorry - had no idea i could still modify the pull request just by pushing new changes to my fork

Just commit the changes and they are included in the pull request.

**Comment by ArminJo** [comment]
Thanks a lot 🥇 
I merged it "manually" it was less effort.

**Pull request closed without merging**